cloudfoundry/get-services.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+function usage {
+cat >&2 <<EOM
+
+Gets all service instances for the given organization GUID and/or service offering
+
+usage: $0 -o [org guid value] -s [service offering name]
+
+Examples:
+  $0 -o org-guid-123 -s aws-elasticsearch
+  $0 -o org-guid-123 -s aws-rds
+  $0 -o org-guid-123 -s aws-elasticache-redis
+  $0 -s custom-domain
+
+EOM
+}
+
+while getopts ":hs:o:" opt; do
+  case ${opt} in
+    h )
+      usage
+      exit 0
+      ;;
+    o )
+      ORGANIZATION_GUID=$OPTARG
```

Comment:
How about making it `-o org_name` then here run `ORGANIZATION_GUID=$(cf org --guid $OPTARG)`

Reply:
Good idea! Changed: 03aeb2f68d22a81dd989ea18d1c6b6f8d08c8380